pages/order-eggs.tsx

```diff
@@ -0,0 +1,12 @@
+const OrderEggs = () => {
+  return (
+    <>
+      <h1>Order Eggs</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+    </>
```

Comment:
nit: extra brackets

Reply:
we need the brackets here to wrap around multiple components